### ISSUE TITLE
index.html in master still want to use bundle.js

### ISSUE THREAD

**Issue by Kailang** [mmontag/dx7-synth-js#1]
The code is amazingly well written and I'm learning it!

_index.html_ in **gh-pages** uses _bundle.js_ and works fine.

However the _index.html_ in **master** still want to use _bundle.js_ which is not yet complied. So, of course, the master branch is broken.

How to make _index.html_ directly use _source code(src/*.js)_ but not _complied code(bundle.js)_?

Cheers,

Kailang

**Comment by Kailang** [comment]
Changing the <script src="src/bundle.js"></script> to <script src="src/app.js"></script> should be the solution. 

And, since dx7-synth-js needs several other libraries(lodash, angular, ngstorage, mml-emitter, midifile, midiplayer) which are not available with the repo, I will include these libraries and try again.

**Comment by mmontag** [comment]
Hi Kailang, thanks for the kind words.
To run locally, you will need to:
- install node.js for your system
- run `npm install` in the project root directory to download dependencies
- run `npm run dev` to compile for development and run a local server

**Comment by Kailang** [comment]
Thanks! It works! Closing the issue!

And... 

I was rewriting my C# Midi library that reads Midi file and synthesizes music. Adding Dx7 style synth has long be a dream of mine!

I've finished porting this to C# this afternoon, but the sound is somehow different from the original js version. I'll rewrite everything(clean up the constants, build interfaces, restructure codes, and more!) before committing the finished version to [Midif](https://github.com/DotLab/Midif).

Cheers.
